docs/licensing-policy.md

```diff
@@ -0,0 +1,47 @@
+# IPFS Licensing Policy
+
+In order to ensure an open source project is actually open and re-usable in a legal sense, it’s critical that licensing information is clear. Repositories in the `ipfs`, `libp2p`, and `ipld` organizations should all follow this policy for licensing their contents.
```

Comment:
Wasn’t sure if `ipfs-shipyard` should be included here, too.

Reply:
Good question. Shipyard includes projects that often originated outside of the org and may have custom licenses due to historical reasons.
For example  [ipfs-shipyard/ipfs-companion](https://github.com/ipfs-shipyard/ipfs-companion#license)  is CC0 (it was originally a hobby project of mine, we've moved it to IPFS Org about a year or two after inception).

Reply:
For what it’s worth, I don’t know if existing repos really need to change (it might be nice to make them all consistent, though). I also definitely hear that a lot of shipyard projects might not start as PL projects, so it seems reasonable that they could have differing licenses. :)